src/components/TxFlow/TxStatusWidget/index.tsx

```diff
@@ -0,0 +1,66 @@
+import { Divider, List, ListItem, ListItemIcon, ListItemText, Paper, Typography } from '@mui/material'
+import CreatedIcon from '@/public/images/messages/created.svg'
+import SignedIcon from '@/public/images/messages/signed.svg'
+import { type TransactionSummary } from '@safe-global/safe-gateway-typescript-sdk'
+import useSafeInfo from '@/hooks/useSafeInfo'
+import { isMultisigExecutionInfo } from '@/utils/transaction-guards'
+import classnames from 'classnames'
+import css from './styles.module.css'
+
+const confirmedMessage = (threshold: number, confirmations: number) => {
+  return (
+    <>
+      Confirmed ({confirmations} of {threshold})
+    </>
+  )
+}
+
+const TxStatusWidget = ({ step, txSummary }: { step: number; txSummary?: TransactionSummary }) => {
+  const { safe } = useSafeInfo()
+  const { threshold } = safe
+
+  const { executionInfo = undefined } = txSummary || {}
+  const { confirmationsSubmitted = 0 } = isMultisigExecutionInfo(executionInfo) ? executionInfo : {}
+
+  const isConfirmedStepIncomplete = step < 1 && !confirmationsSubmitted
+
+  return (
+    <Paper>
+      <div className={css.header}>
+        <img src="/images/logo-no-text.svg" alt="Safe logo" width="32px" />
+        <Typography variant="h6" fontWeight="700">
+          Transaction status
+        </Typography>
+      </div>
+      <Divider />
+      <div className={css.content}>
```

Comment:
Also here.
```suggestion
      </div>

      <Divider />

      <div className={css.content}>
```

Reply:
Ah got it! I will mind it for future reference